daemon/src/projection.rs

```diff
@@ -132,6 +132,17 @@ pub struct Cfd {
     #[serde(with = "round_to_two_dp")]
     pub initial_price: Price,
 
+    /// Sum of all costs
+    ///
+    /// Includes the opening fee and all funding fees period that were already charged.
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
+    pub accumulated_funding_fees: Amount,
+
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
+    pub funding_fee_per_period: Amount,
```

Comment:
We should use `round` here instead of `period`. I think we agreed on round, introducing `period` is yet another term.

Reply:
renamed.